### DISCUSSION TITLE
Support to link third party .obj files in final exe

### DISCUSSION THREAD

**Discussion by YogiYang** [twinbasic/twinbasic#400] [General]
Are there any plans to support linking .obj files compiled in C or FPC in final EXE?

**Comment by Kr00l** [comment]
There is already issue #120 for that topic.

**Comment by WaynePhillipsEA** [comment]
Hi, and thanks for your question.  As @Kr00l mentioned, there is an open issue to discuss static linking.  Linking to OBJ files would be relatively straight-forward, but will require syntax support (similar to Declare statements, but working with raw symbols) in order to allow the compiler to make use of them.
